GameData/SPEngine/Parts/RD100_proc.cfg

```diff
@@ -1,26 +1,28 @@
-+PART[RO-RealEngines-RD-100]:NEEDS[!ROEngines]:AFTER[ZREALPLUME]
++PART[RO-RealEngines-RD-100]:NEEDS[!ROEngines]:FOR[SPEngine]
 {
 	@name = SPEngineRD100
 	@title = a-class Procedural Engine
 	@description = Russian Alcohol/LOx atmospheric engine, like RD-100 series.
+	@TechRequired = unlockParts
 	MODULE
 	{
 		name = ModuleSPEngine
-		familyLetter = a
+		familyLetter = A
 	}
 	!identicalParts =
 	!engineType = // prevent ROE from deleting us
 }
 
-+PART[ROE-RD100]:NEEDS[ROEngines]:AFTER[ZREALPLUME]
++PART[ROE-A4]:NEEDS[ROEngines]:FOR[SPEngine]
```

Comment:
No.  The A-4 is not an a-class engine (Russian alcolox), it's an A-class engine (American alcolox), they're different classes with different characteristics and should keep having different models.

Reply:
the ROE A4 uses 
model = ROEngines/Assets/RealEngines/RD_100

Reply:
Wat.

Why doesn't ROE use the Taerobee A4 model?  Licence incompatibilities?

Sigh.  Well, whatever you come up with, make sure that the A-class (A4) and a-class (RD100) engines have visually distinct models.  (I'd be in favour of sticking with Taerobee for the A-4 even when ROE is installed.)

Reply:
not sure it could be a licence issue